src/workerd/io/worker-interface.capnp

```diff
@@ -482,6 +488,16 @@ struct JsValue {
   }
 }
 
+interface AbortSignal $Cxx.allowCancellation {
+    struct Params {
```

Comment:
Nitpick: Indentation is off here.

Reply:
Is there a capnp autoformatter? I would love to put once in CI if it exists.

Reply:
Sorry, there isn't.